gpAux/gpdemo/README

```diff
@@ -44,33 +40,38 @@ RUNNING GP DEMO
    The "." will source the greenplum_path.sh into the current
    shell, instead of starting a subprocess.
 
-4. Create the cluster:
+3. Create the cluster at the current directory:
+
+	gpdemo
 
-	make cluster
+	By default, all data of the demo cluster will be stored in
+	"./datadirs" of the current directory. You may change this by
+	cd to another directory or using "DATADIRS" environment variable
+	described in the next section.
 
-5. Source gpAux/gpdemo/gpdemo-env.sh
+4. Source ./gpdemo-env.sh
 
-6. If setup completes correctly, you will have a virtual Cloudberry
+5. If setup completes correctly, you will have a virtual Cloudberry
    Database system running on the local host. You can connect to it
    using the psql client program as follows:
 
 	psql postgres
 
-7. To exit the psql client program:
+6. To exit the psql client program:
 
 	\q
 
-8. See the Cloudberry Database User Guide for more information on
+7. See the Cloudberry Database User Guide for more information on
    using your Cloudberry Database system.
 
 
 ******************************************************************
 REMOVING GP DEMO
 ******************************************************************
 
-1. From the gpdemo directory run make clean:
+1. From the gpdemo directory run "gpdemo -c":
 
-	make clean
+	gpdemo -d
```

Comment:
-c or -d?

Reply:
I think the meaning is that execute `gpdemo -d` in the directory where `gpdemo` gets executed. I'll rewrite this sentence.